www/_template/reference/plugins.md

```diff
@@ -48,9 +48,15 @@ Use this hook to read or make changes to the completed Snowpack configuration ob
 
 Load a file from disk and build it for your application. This is most useful for taking a file type that can't run in the browser (TypeScript, Sass, Vue, Svelte) and returning JS and/or CSS. It can even be used to load JS/CSS files directly from disk with a build step like Babel or PostCSS.
 
+Parameters:
+- `filePath` | `string` : the raw path of the file?
+- `isSSR`| `boolean` | Default: `false` : is it SSR? IDK
+- `isDev`| `boolean` | Default: `false` : ??
+- `fileExt` | `string` : the file extension?
+
 #### transform()
 
-Transform a file's contents. Useful for making changes to all types of build output (JS, CSS, etc.) regardless of how they were originally loaded from disk.
+Transforms build output (JS, CSS, etc.) regardless of how they were originally loaded from disk.
```

Comment:
I know you were just improving the wording that was already here, but maybe we should avoid saying “transform” to describe the hook with the same name? It’s all kinda “transforming” in a way.

I wonder if it’s more helpful to say something about how `load()` is the first stage of modifying code, and `transform()` is the 2nd.

Load has to deal with every possible file type, whereas transform will only deal with web-safe extensions (`.html`, `.css`, `.js`, and web assets like fonts & images).